app/assets/javascripts/discourse/tests/unit/models/topic-tracking-state-test.js

```diff
@@ -457,7 +457,7 @@ discourseModule("Unit | Model | topic-tracking-state", function (hooks) {
         trackingState.onMessageIncrement(() => {
           messageIncrementCalled = true;
         });
-        publishToMessageBus(`/unread/${currentUser.id}`, unreadTopicPayload);
+        publishToMessageBus("/topic-tracking-state", unreadTopicPayload);
```

Comment:
It doesn't seem right to me that we seem to have lost the user for the unread channel, shouldn't that still be there?